tsconfig.json

```diff
@@ -28,7 +28,7 @@
     "noUnusedLocals": true,
     "strictFunctionTypes": true,
     "alwaysStrict": true,
-    "strictPropertyInitialization": true,
+    "strictPropertyInitialization": false, // to support Object.assign in the constructor of a class
```

Comment:
Where do we do this and can we find another way? :) It's a helpful linting rule.

Reply:
I'm doing `Object.assign(this, config)` within the Icon's constructor.

Reply:
Fixed.